### ISSUE TITLE
Is there a way to move the Manipolator icons to a toolbar?

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#103]
![capture](https://cloud.githubusercontent.com/assets/11047164/13534413/448832f2-e235-11e5-8d1c-fd1d46633bd8.PNG)

Well... its a mess IMHO :-1:

**Comment by nilsschmidt1337** [comment]
![capture](https://cloud.githubusercontent.com/assets/11047164/13534611/490a0836-e236-11e5-845f-f7b1e5c0dc4b.PNG)
Second variation...

**Comment by nilsschmidt1337** [comment]
I will provide a solution with a script for custom GUI designs. A drag and drop solution (e.g. a CoolBar) seems to be more complicated and error prone.

**Comment by nilsschmidt1337** [comment]
I created a configuration file parser.
The parser is able to read
**layout_3D_editor.cfg** (@ application directory); contents of this file:
```
POSSIBLE_VALUES      ; (NORTH|EAST|WEST); (HORIZONTAL|VERTICAL|DROPDWNMNU); (NO_LABEL|Any other text...) ; // A short list of all possible values. Not every feature is supported on every tool item!
SYNC_AND_RECENT_FILES; NORTH; HORIZONTAL; NO_LABEL; // Tool item for synchronisation and recent files
PROJECT_MANAGEMENT   ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for project actions (save, load, ...)
OPEN_SAVE_DATFILE    ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for datfile actions (create, load)
SHOW_AND_HIDE        ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for Show All/Hide
MISC_TOGGLE          ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for "Move Adjacent Data", ...
UNDO_REDO            ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for Undo/Redo 
TRANSFORMATION_MODE  ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for "Select", "Move", "Rotate", ...
MANIPULATOR_MODE     ; NORTH; HORIZONTAL; NO_LABEL; // Tool item to switch between local and global mode 
MANIPULATOR_ACTIONS  ; NORTH; DROPDWNMNU; NO_LABEL; // Tool item for manipulator actions (as drop-down menu)
MISC_CLICK           ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for rounding, BFC,... (fixed layout )
CUT_COPY_PASTE_DELETE; NORTH; HORIZONTAL; NO_LABEL; // Tool item for clipboard actions
OBJECT_MODE          ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for the obj. mode selector (lines, surfaces..)
INSERT_AT_CURSOR     ; NORTH; HORIZONTAL; NO_LABEL; // Tool item to toggle "Insert at cursor"
ADD_SOMETHING        ; NORTH; HORIZONTAL; NO_LABEL; // Tool item to add objects
COLOUR_BAR           ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for the colour bar 
LINE_THICKNESS       ; NORTH; HORIZONTAL; NO_LABEL; // Tool item for different line sizes
```
and places the toolbars according to the order and the drawing style. Currently, the manipulator actions  are the only component with "DROPDWNMNU" (drop-down menu) support.

**Comment by nilsschmidt1337** [comment]
Closed with commit 0565529.

**Comment by nilsschmidt1337** [comment]
Forgot to update **layout_3D_editor.cfg**...

**Comment by nilsschmidt1337** [comment]
Finally, closed with commit c1f1f58.
